src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -87,6 +87,15 @@ def _upsert(parent, collection_name, obj_to_add, key_name):
 
     collection.append(obj_to_add)
 
+
+def _get_default_value(balancer, property_name, option_name):
+    values = [x.name for x in getattr(balancer, property_name)]
+    if len(values) > 1:
+        raise CLIError("Multiple possible values found for '{0}': {1}\nSpecify '{0}' "
+                       "explicitly.".format(option_name, ', '.join(values)))
+    return values[0]
```

Comment:
If the property doesn't exist, then we should return raise here for arg value, rather has the indexError